platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/ECMSSearchWebContentWithAccentedCharactersTestIT.java

```diff
@@ -0,0 +1,87 @@
+package org.exoplatform.platform.qa.ui.platform.ecms;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Configuration;
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.ecms.pageobject.CreateNewDocument;
+import org.exoplatform.platform.qa.ui.ecms.pageobject.SiteExplorerHome;
+import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.exoplatform.platform.qa.ui.selenium.platform.NavigationToolbar;
+import org.exoplatform.platform.qa.ui.selenium.platform.administration.ContentAdministration;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selectors.*;
+import static com.codeborne.selenide.Selenide.$;
+import static org.exoplatform.platform.qa.ui.core.PLFData.password;
+import static org.exoplatform.platform.qa.ui.core.PLFData.username;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ecms.ECMSLocator.ELEMENT_ICON_SEARCH;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ecms.ECMSLocator.ELEMENT_SEARCH_BTN;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ecms.ECMSLocator.ELEMENT_TITLE_WEB_CONTENT;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+import static org.exoplatform.platform.qa.ui.selenium.testbase.LocatorTestBase.ELEMENT_SKIP_BUTTON;
+
+/**
+ * Created by esmaoui on 29/03/2018.
+ */
+
+
+@Tag("sniff")
+@Tag("ecms")
+public class ECMSSearchWebContentWithAccentedCharactersTestIT extends Base {
+
+    ContentAdministration contentAdministration;
+
+    NavigationToolbar navigationToolbar;
+
+    SiteExplorerHome siteExplorerHome;
+    CreateNewDocument createNewDoc;
+
+    ManageLogInOut manageLogInOut;
+
+    @BeforeEach
+  public void setupBeforeMethod() {
+            info("Start setUpBeforeMethod");
+
+               contentAdministration = new ContentAdministration(this);
+              navigationToolbar = new NavigationToolbar(this);
+               siteExplorerHome = new SiteExplorerHome(this);
+               createNewDoc= new CreateNewDocument(this);
+               manageLogInOut=new ManageLogInOut(this);
+                 if ($(ELEMENT_SKIP_BUTTON).is(Condition.exist)) {
+                     $(ELEMENT_SKIP_BUTTON).click();
+                   }
+        manageLogInOut.signInCas(username, password);
+}
+
+     /*
+      bug ECMS-7688
+     */
```

Comment:
Wrong format for comments on method, and we should add a description to directly see what this test is about (without going to jira).

Reply:
done